blockprotocol/blockprotocol
# Update blockprotocol.org to 0.2

This PR updates the `site` package to use the new BP approach introduced in #323. 

This changes:
1. Move metadata types from `blockprotocol` to `@blockprotocol/core` and update imports. Also update where the public JSON schema file (used by SchemaStore) is generated from
2. Update the data provided to blocks to use the new structure
3. Update function signatures and types around entity editing
4. Filter out blocks from the hub which are < 0.2
5. Add the Code block as a demo, from https://github.com/hashintel/hash/pull/623

## How to test

1. Check the schema editor still works
2. Check the Code block in the Hub is the only one that displays, and functions correctly

## Next steps

Next introduced to the `v0.2` branch will be the actual new specification / documentation.
